kubernetes_asyncio/config/__init__.py

```diff
@@ -12,9 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from os.path import exists, expanduser
+
 from .config_exception import ConfigException
 from .incluster_config import load_incluster_config
 from .kube_config import (
-    list_kube_config_contexts, load_kube_config, load_kube_config_from_dict,
-    new_client_from_config, new_client_from_config_dict, refresh_token,
+    KUBE_CONFIG_DEFAULT_LOCATION, list_kube_config_contexts, load_kube_config,
+    load_kube_config_from_dict, new_client_from_config,
+    new_client_from_config_dict,
 )
+
+
+async def load_config(**kwargs):
+    """
+    Wrapper function to load the kube_config.
+    It will initially try to load_kube_config from provided path,
+    then check if the KUBE_CONFIG_DEFAULT_LOCATION exists
+    If neither exists, it will fall back to load_incluster_config
+    and inform the user accordingly.
+
+    :param kwargs: A combination of all possible kwargs that
+    can be passed to either load_kube_config or
+    load_incluster_config functions.
+    """
+    if "kube_config_path" in kwargs.keys() or exists(expanduser(KUBE_CONFIG_DEFAULT_LOCATION)):
+        await load_kube_config(**kwargs)
+    else:
+        print(
```

Comment:
I think this print() could be annoying if you run your code as a local script and as a pod in a cluster. 
We can:
1. remove it, ;)
2. use logging.debug instead
3. print only if load_incluster_config failed.

what do you think?

Reply:
Given my experience with the same print message from the original `kubernetes` package, I certainly agree it can be annoying lol. I just erred on the conservative side with the changes I made to the original code. What are your thoughts on using a warning instead? That way, it only prints once.

Reply:
Sure, a warning will be good :+1: